webpack.demo.js

```diff
@@ -0,0 +1,54 @@
+var webpack = require('webpack');
+var HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: {
+    'angular2': [
+      'zone.js',
+      'reflect-metadata',
```

Comment:
I'm not sure how much the caching will be an issue - in practice, the releases won't be *that* frequent, will they? Caching will take over in the browser after the first time they hit the new page...caching external dependencies separately via script tag has the advantage of displaying a good practice for end users, but as far as this project goes, it doesn't really matter too much IMO.

The avoiding unnecessary webpack work is a valid point though - decreasing build tool runtime has productivity ramifications.

I'm indifferent about choosing one approach over the other, so I'm fine with either as long as the one with the better arguments win out.